src/Misc/externals.sh

```diff
@@ -180,3 +184,8 @@ if [[ "$PACKAGERUNTIME" == "linux-arm" ]]; then
     acquireExternalTool "$NODE_URL/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-armv7l.tar.gz" node fix_nested_dir
     acquireExternalTool "$NODE_URL/v${NODE10_VERSION}/node-v${NODE10_VERSION}-linux-armv7l.tar.gz" node10 fix_nested_dir
 fi
+
```

Comment:
Ideally, we could set some kind of environment variable in our workspace to get the tasks.zip that is being used in production (ring0). When we are trouble shooting a problem. But this is great!